src/logging.rs

```diff
@@ -114,8 +118,9 @@ pub fn init(log_config: LoggingConfig) {
             .with_span_list(true)
             .with_file(true)
             .with_line_number(true)
+            .with_ansi(log_config.with_ansi)
             .boxed(),
-        LogFormat::Text => subscriber.compact().boxed(),
+        LogFormat::Text => subscriber.compact().with_ansi(log_config.with_ansi).boxed(),
```

Comment:
Another approach would be to have `text` (with-ansi) and `plaintext` (or a similar name) that doesn't feature ansi formatting.